src/core_plugins/kibana/public/discover/index.html

```diff
@@ -25,25 +25,19 @@
       >
         <div class="typeahead" kbn-typeahead="discover">
           <div class="kuiLocalSearch">
+            <div class="kuiLocalSearch__icon kuiIcon fa-search"></div>
             <input
               parse-query
               input-focus
               kbn-typeahead-input
               ng-model="state.query"
-              placeholder="Search..."
+              placeholder="Search... ex. status:200 AND extension:PHP"
```

Comment:
Can we use "e.g." instead of "ex."? I think this is more common, and it's also used in our writing style guide.